RAKE/RAKE.py

```diff
@@ -27,16 +27,42 @@ def FoxStopList():
     return FoxStopList.words()
 
 
-def load_stop_words(stop_word_file):
+def MySQLStopList():
+    from .stoplists import MySQLStopList
+    return MySQLStopList.words()
+
+
+def NLTKStopList():
+    from .stoplists import NLTKStopList
+    return NLTKStopList.words()
+
+
+def load_stop_words(stop_word_file, divide):
+    """
+    Utility function to load stop words from a file and return as a list of words
+    @param stop_word_file Path and file name of a file containing stop words.
+    @return list A list of stop words.
+    """
+    stop_words = []
+    for line in open(stop_word_file):
+        if divide:
+            for word in line.split():  # in case more than one per line
+                stop_words.append(word)
+    return stop_words
+
+def load_stop_words_delimiter(stop_word_file, delimiter):
```

Comment:
I would unify the logic here with the logic in `load_stop_words`. Seems like load_stop_words is just a specific case of this function.